features/networking/ipv6_dualstack.feature

```diff
@@ -80,3 +79,35 @@ Feature: ipv6 dual stack cluster test scenarios
     Then the step should fail
 
 
+  # @author zzhao@redhat.com
+  # @case_id OCP-46816
+  @admin
+  @network-ovnkubernetes
+  @4.10 @4.9
+  @baremetal-ipi
+  Scenario: ipv6 for nodeport service
+    Given I store the workers in the :workers clipboard
+    And the Internal IPv6 of node "<%= cb.workers[0].name %>" is stored in the :worker0_ip clipboard
+    Given I have a project
+    And evaluation of `rand(30000..31000)` is stored in the :port clipboard
+    Given I obtain test data file "networking/nodeport_test_pod.yaml"
+    When I run the :create client command with:
+      | f | nodeport_test_pod.yaml |
+    Then the step should succeed
+    When I obtain test data file "networking/nodeport_test_service.yaml"
+    When I run oc create over "nodeport_test_service.yaml" replacing paths:
+      | ["spec"]["ports"][0]["nodePort"] | <%= cb.port %> |
+      | ["spec"]["ipFamilyPolicy"] | "RequireDualStack" |
```

Comment:
Existing networking/nodeport_test_service.yaml do not have ipFamilyPolicy option, should you include new updated networking/nodeport_test_service.yaml in this PR?

Reply:
yes, replace can add this option